src/omnisharp/loggingEvents.ts

```diff
@@ -151,6 +150,10 @@ export class DocumentSynchronizationFailure implements BaseEvent {
     constructor(public documentPath: string, public errorMessage: string) { }
 }
 
+export class OpenURL {
+    constructor(public url: string) { }
```

Comment:
Why not require this to be a VSCode.Uri?

Reply:
Vscode.Uri is a class and I am not sure how to create a fake for it for testing purpose. Is it not acceptable this way ?